views/default/js/elgg/process_deferred.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Processes functions queued by elgg_defer() and defines it.
+ *
+ * @internal Do not alter or use this view.
+ *
+ * @note Early in the boot sequence elgg_defer is defined in the view "page/elements/head"
+ */
+if (0) { ?><script><?php } ?>
```

Comment:
Make this a plain `.js` file: `process_deferred.js` instead of `process_deferred.php`.

Maybe even makes sense to define as an AMD module?